web_src/js/index.js

```diff
@@ -179,4 +180,6 @@ onDomReady(() => {
   initRepoDiffView();
   initPdfViewer();
   initScopedAccessTokenCategories();
+
+  initLoadBranchesAndTagsButton();
```

Comment:
I think only some pages needs this "init". Is there any existing "init" functions which already checks whether current page is a "commit view" page? Then this "init" could be called there.

Reply:
I don't think there is yet.
At the moment, everything is most likely registered in `index.js`

Reply:
Yes, we don't have such a concept so far. We just call all `init`s which is usually not an issue as long as the functions return quickly.